src/components/UserComponents/types.tsx

```diff
@@ -0,0 +1,5 @@
+export const enum SizeType {
```

Comment:
💯

Reply:
❓ Is this an enum that can be used for component sizes throughout the app or is it restricted to UserComponents?

Reply:
good point, changed it to fit with chakra's naming style of sm, md, lg so can be used throughout the app and so i can use it for avatar size

Reply:
Can you also move it up to be under src? (either in constants.ts or a new types.ts is prob good) That way it’s easier for others to find + reuse

Reply:
Yeah good point, moved to constants.ts